webdriver-spec.html

```diff
@@ -2049,6 +2049,13 @@
   <a>getting a property</a> named <code>url</code>
   from the <var>parameters</var> argument.
 
+ <li><p>If <var>url</var> is a malformed URL,
```

Comment:
Yay github for losing context...

A fragment must be 0 or more Url units (see [url spec](https://url.spec.whatwg.org/#syntax-url-fragment). From the URL spec `A URL
must be either a relative URL with fragment or an absolute URL with fragment`. Since we only want the the absolute url we should describe it as such.